helper.fish

```diff
@@ -1383,8 +1383,9 @@ function checkLogId
   or begin popd; return 1; end
 
   set -l ids (find lib arangod arangosh enterprise -name "*.cpp" -o -name "*.h" \
-    | xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\)("[a-f0-9]*"' \
-    | sed -e 's:^.*LOG_[^(]*("\([a-f0-9]*\)".*:\1:')
+    | xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\)("[a-z0-9]*"' \
```

Comment:
Would advise to use `| xargs grep -h 'LOG_\(TOPIC\|TRX\|TOPIC_IF\)("[^\"]*"' | \` here.